src/schemas/chat.py

```
@@ -1,20 +1,44 @@
+- 타입이 맞지 않으면 자동으로 에러를 발생시킵니다.
+  예: country_id에 문자열 "abc"가 오면 → 422 Unprocessable Entity 에러
+- FastAPI와 연동되어 Swagger 문서(/docs)에 자동으로 요청/응답 형식이 표시됩니다.
+
+[사용되는 곳]
+- ChatRequest: POST /api/v1/chat, POST /api/v2/chat의 요청 본문(body)
+- ChatResult: CommonResponse의 result 필드에 들어가는 응답 데이터
+"""
+
 from pydantic import BaseModel
-from typing import List
+from typing import List, Optional
```

Ruff (0.15.2)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)